delivery_state/migrations/13.0.2.0.0/post-migrate.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """We'll be using the former template in this module"""
+    delivery_template_id = env.ref("delivery_state.delivery_notification").id
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE res_company SET
```

Comment:
This will fail if you are migrating from 12.0. You have to check for the existence of the column (`column_exists` method in openupgradelib).

Reply:
fixed